event-handler-template-examples/src/main/java/no.unit.nva.events.examples/DestinationsEventConsumer.java

```diff
@@ -0,0 +1,37 @@
+package no.unit.nva.events.examples;
+
+import com.amazonaws.services.lambda.runtime.Context;
+import no.unit.nva.events.handlers.DestinationsEventBridgeEventHandler;
+import no.unit.nva.events.models.AwsEventBridgeDetail;
+import no.unit.nva.events.models.AwsEventBridgeEvent;
+import nva.commons.utils.JacocoGenerated;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Demo class. Will be deleted.
+ */
+public class DestinationsEventConsumer
+    extends DestinationsEventBridgeEventHandler<DataciteDoiRequest, DataciteDoiRequest> {
+
+    private static final Logger logger = LoggerFactory.getLogger(DestinationsEventConsumer.class);
+
+    @JacocoGenerated
+    public DestinationsEventConsumer() {
+        super(DataciteDoiRequest.class);
+    }
+
+    @JacocoGenerated
+    @Override
+    protected DataciteDoiRequest processInputPayload(
+        DataciteDoiRequest input,
+        AwsEventBridgeEvent<AwsEventBridgeDetail<DataciteDoiRequest>> event,
```

Comment:
this is the whole event in case someone needs some extra information. The event input is found also in the parameter file